ansible/roles/jupyterhub/files/jupyterhub_config.py

```diff
@@ -240,3 +240,21 @@
 ##########################################
 # END CUSTOM DOCKERSPAWNER
 ##########################################
+
+# Dynamic config to setup new courses
+
+# course setup service name
+service_name = os.environ.get('DOCKER_SETUP_COURSE_SERVICE_NAME', 'setup-course')
```

Comment:
Our existing code uses the `or` operator to set defaults, i.e. `os.environ.get('DOCKER_SETUP_COURSE_SERVICE_NAME') or 'setup-course'` let's change all the environ.get's to use this convention.

Reply:
agree